docs/advanced-topics/ideal_integration.md

```diff
@@ -6,21 +6,25 @@ nav_order: 1
 permalink: /advanced-topics/ideal-integration/
 ---
 
-# Ideal integration with OSS-Fuzz 
-OSS projects have different build and test systems. So, we can not expect them
-to have a unified way of implementing and maintaining fuzz targets and
-integrating them with OSS-Fuzz. However, we will still try to give
-recommendations on the preferred ways.
+# Ideal integration with OSS-Fuzz
+{: .no_toc}
 
-Here are several features (starting from the easiest) that will make automated
-fuzzing simple and efficient, and will allow to catch regressions early on in
-the development cycle. 
+OSS projects have different build and test systems. We can't expect them all to
+implement and maintain fuzz targets or integrate them with OSS-Fuzz in the same
+way. However, we do have recommendations.
+
+This page documents several features (starting from the easiest) that will make
+automated fuzzing simple and efficient, and will help you catch regressions
+early in the development cycle. This simple
+[example](https://github.com/google/oss-fuzz/tree/master/projects/example/my-api-repo)
+covers most of the items. 
 
 - TOC
 {:toc}
 ---
 
-## TL;DR
+## TL;DR 
```

Comment:
Is `TL;DR` ok for documentation? Should we consider using `Summary` instead?

Reply:
Good point, you're right. Changed to Summary.